cypress/support/fragments/data_import/match_profiles/newMatchProfile.js

```diff
@@ -0,0 +1,99 @@
+import {
+  Button,
+  Select,
+  TextField,
+  SelectionList,
+  Accordion,
+  SelectionOption
+} from '../../../../../interactors';
+import { getLongDelay } from '../../../utils/cypressTools';
+
+const criterionValueTypeList = SelectionList({ id: 'sl-container-criterion-value-type' });
+const criterionValueTypeButton = Button({ id:'criterion-value-type' });
+const optionsList = {
+  holdingsHrid: 'Admin data: Holdings HRID',
+  itemHrid: 'Admin data: Item HRID',
+  pol: 'Acquisitions data: Purchase order line (POL)'
+};
+
+const fillExistingRecordFields = ({ field, in1, in2, subfield }) => {
+  if (field) {
+    cy.do(TextField({ name: 'profile.matchDetails[0].existingMatchExpression.fields[0].value' }).fillIn(field));
+  }
+  if (in1) {
+    cy.do(TextField({ name: 'profile.matchDetails[0].existingMatchExpression.fields[1].value' }).fillIn(in1));
+  }
+  if (in2) {
+    cy.do(TextField({ name: 'profile.matchDetails[0].existingMatchExpression.fields[2].value' }).fillIn(in2));
+  }
+  if (subfield) {
+    cy.do(TextField({ name: 'profile.matchDetails[0].existingMatchExpression.fields[3].value' }).fillIn(subfield));
```

Comment:
```
function fillExistingRecordFields(value, selector) => {
  const map = {
    field: 'profile.matchDetails[0].existingMatchExpression.fields[0].value',
    in1: 'profile.matchDetails[0].existingMatchExpression.fields[1].value',
    in2: 'profile.matchDetails[0].existingMatchExpression.fields[2].value',
    subfield: 'profile.matchDetails[0].existingMatchExpression.fields[3].value'
  }

  cy.do(TextField({ name: map[selector]).fillin(value)
```
You can call function like this fillExistingRecordFields('stringForFilling', 'in1')

Don't use in function more than one if please

Reply:
Done